laravelServer/server/resources/views/welcome.blade.php

```diff
@@ -154,6 +154,26 @@
                                 </div>
                             </div>
                         </div>
+						
+						<div class="p-6 border-t border-gray-200 dark:border-gray-700 md:border-l">
+                            <div class="flex items-center">
+                                <form method="post" action="{{ route('options') }}">
+                                    <div class="onoffswitch">
+                                        {{ csrf_field() }}
+                                        <input type="checkbox" name="status" class="onoffswitch-checkbox" id="placeholder7" tabindex="0" onchange="this.form.submit()" {{ $statuses->where('type', 6)->where('on', true)->isNotEmpty() ? 'checked' : '' }}>
```

Comment:
also {{ $statuses->where('type', 6)->where('on', true)->isNotEmpty() ? 'checked' : '' }} decides whether or not a status button is shown as on based on actual database state. If you want to make it represent type 7 (zuil) it should probably check for where('type', 7).